egs/hkust/s5b/local/hkust_data_prep.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
+
+. path.sh
```

Comment:
or alternatively, do
set -e
above this, which may help in error checking in the rest of the file (you don't seem to be doing '|| exit 1' anywhere).